LICENSE.md

```diff
@@ -1,20 +1,57 @@
-### WordPress - Web publishing software
+## Gutenberg
 
     Copyright 2011-2021 by the contributors
 
-This program is free software; you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation; either version 2 of the License, or
-(at your option) any later version.
+**License for Contributions (on and after April 15, 2021)**
 
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
+All code contributed to the Gutenberg project is dual-licensed, 
+and released under both of the following licenses: 
 
-You should have received a copy of the GNU General Public License
-along with this program; if not, write to the Free Software
-Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+the GNU General Public License as published by the Free Software Foundation; 
+either version 2 of the License or (at your option) any later version (the “GPL”)
```

Comment:
WDYT about putting the whole `either version 2 of the License or (at your option) any later version (the “GPL”)` in parenthesis? I had trouble reading the sentence since I was expecting "2 things" (based on the previous "both of the following licenses) but the "either" refers to many versions of GPL itself, plus the MPL afterwards.

Reply:
I don't feel super-strongly, but I don't think I like that better. The relevant version(s) of the GPL feels like it is important enough that it shouldn't be relegated to a parenthetical to me. Also, if we do the parenthetical then we have a kinda awkward double parenthetical with `(the "GPL)`, giving us either `(...any later version (the "GPL"))` or, more likely `(...any later version) (the "GPL")`.

Like I said, I don't feel strongly here, so feel free to push back. 🙂

Reply:
Far from strong feelings here from my side. Let's leave it as is and we can always refine if need arises 👍